specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,118 @@
+# Service Subcategory
+
+The Service Subcategory is a further classification of the Service Category for a [*service*](#glossary:service) based on its core function.  The Service Subcategory (in conjunction with the Service Category) is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.  
+
+The ServiceSubcategory column MUST be present in a FOCUS dataset and MUST NOT be null. This column is of type String and MUST be one of the allowed values.  Each Service Subcategory value MUST have one and only one Service Category parent. Though a given *service* can have multiple purposes, each *service* SHOULD have one and only one Subcategory that best aligns with its primary purpose. 
+
+## Column ID
+
+ServiceSubcategory
+
+## Display Name
+
+Service Subcategory
+
+## Description
+
+Further classification of the Service Category for a *service* based on its core function.
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column type     | Dimension      |
+| Feature level   | Mandatory      |
+| Allows nulls    | False          |
+| Data type       | String         |
+| Value format    | Allowed Values |
+
+Allowed values:
```

Comment:
Can we change this to be a separate table for each service category? The Unit Format attribute does this. The empty cells and rows feel like added noise and are making this harder to review, given how big the list is.

I'm going to hold off on this PR for the time being as I have many comments and don't want to reapply them later.

Reply:
I'm happy to eliminate the blank rows between Categories, which I have now done.  (I found it to be slightly more readable, but I can totally see that not everyone may feel that way, and it was a non-standard composition anyway.)

I'm more hesitant to break the Categories into separate tables, given that the column composition for all rows is the same (unlike Unit Format, where the column headers differ).  Also, I believe it's useful to keep all the possible allowed values in one table for the purposes of spec maintainability, provider utility, and practitioner reference.  I would prefer to discuss further if you feel this needs to happen.

Reply:
I tend to agree that without empty rows but keeping in one table seems like the right approach. Is there any value in populating the service category for each row rather than only on the first row and leaving the rest empty? If it doesn't help, ignore the suggestion.

Reply:
Considering that I don’t know how it will be formatted (in terms of page breaks) in the PDF version, I lean towards providing the Service Category in each row, but I don’t feel strongly about it.

Reply:
Splitting them is about making it easier to comprehend. There's no value with a massive table in a PDF file. Smaller tables, regardless of column alignment is easier to work with. Regardless, this isn't accessible, so at a bare minimum, the first column has to be populated for every row.

Reply:
I have now added the Service Category to every Service Subcategory row.  Please confirm that this is sufficient for approval!

Reply:
Adding 20 (and in the future a lot more) tables doesn't seem like a good idea. A single table may not be ideal but it seems like the best approach. We can revisit in the future if there's a compelling reason.

Reply:
Resolving this comment per the above.  Will leave as-is and address in a future consistency PR.